internal/cli/atlas/quickstart/dbuser_setup.go

```diff
@@ -114,3 +115,10 @@ func generatePassword() (string, error) {
 
 	return pwd, nil
 }
+
+func getStringPointerIfNotEmpty(input string) *string {
```

Comment:
Is that duplicated with global GetStringPointerIfNotEmpty?

Reply:
Yep thanks for pointing that out. I'll remove the duplicates (initially couldn't decide where to put it)